packages/components/loading-spinner/src/loading-spinner.tsx

```diff
@@ -27,6 +27,11 @@ const sizePerScale = {
 const positionOrigin = '20px';
 
 type TLoadingSpinnerProps = {
+  /**
+   * Set the amount of time to delay the loading spinner before it renders.
+   * The default value of is 1000ms.
+   */
+  maxDelayDuration: number;
```

Comment:
This should be optional.

```suggestion
  maxDelayDuration?: number;
```